.husky/pre-commit

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
+export NVM_DIR="$HOME/.nvm"
```

Comment:
not to sure if this would work on machines without nvm installed, maybe someone has a better idea

Reply:
No, the fix I added does not require it to be present anymore

It was previously but now if it can't find nvm it just executes npx lint-staged without changing the node version first

Reply:
Gotcha, that's what I assumed but just wanted to make sure. Thanks!